runtime/src/main/java/org/corfudb/runtime/collections/LrMultiStreamMergeStreamListener.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.runtime.collections;
+
+/**
+ * This is the callback interface that any client subscribing to CorfuStore updates must implement.
+ *
+ * This interface sees ordered logs from :
+ * 1. client-streams from client-Namespace, and,
+ * 2. LrStatusTable from corfuSystem-Namespace.
```

Comment:
Wouldnt it be better to keep this system table generic and not necessarily the status table?  A wrapper to the client is provided and internally our code subscribes to the status table or another table depending on the requirement.